client/my-sites/plan-features/index.jsx

```diff
@@ -114,7 +114,7 @@ export class PlanFeatures extends Component {
 							initialSelectedIndex={ initialSelectedIndex }
 						>
 							<table className={ tableClasses }>
-								<caption class="screen-reader-text">
+								<caption className="plan-features__screen-reader-text screen-reader-text">
```

Comment:
Where did `plan-features__screen-reader-text` style come from? It's not in the master.

Reply:
Thanks for approving @donpark !

It was to prevent the linter from complaining. CSS rules are supposed to be prefixed with the name of the directory.

Reply:
aha. maybe I'll try that trick when I get tired of linter complaining. 👍

Reply:
I guess I could just update the class name in the scss file, and remove the old class name

Reply:
I'm not fussy. Anything or nothing is fine by me. :-)